DependencyInjection/TrikoderOAuth2Extension.php

```diff
@@ -48,7 +48,7 @@ public function load(array $configs, ContainerBuilder $container)
         $this->configureResourceServer($container, $config['resource_server']);
         $this->configureScopes($container, $config['scopes']);
 
-        $container->getDefinition('trikoder.oauth2.event_listener.authorization.convert_to_response')
+        $container->getDefinition('Trikoder\Bundle\OAuth2Bundle\EventListener\ConvertExceptionToResponseListener')
```

Comment:
@rjwebdev You can use now in the entire extension the `::class` notation instead of a string like this.

Reply:
Good point! Wanted to be to fast, will replace it

Reply:
@rjwebdev You seemed to have missed this one.